addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -287,7 +291,7 @@
 			<animation effect="fade" time="150">VisibleChange</animation>
 			<animation effect="slide" start="0,0" end="0,80" time="0" condition="![VideoPlayer.HasSubtitles + VideoPlayer.SubtitlesEnabled]">Conditional</animation>
 			<animation effect="slide" start="0,0" end="0,40" time="0" condition="!VideoPlayer.HasSubtitles">Conditional</animation>
-			<animation effect="slide" start="0,0" end="55,0" time="0" condition="VideoPlayer.Content(LiveTV)">Conditional</animation>
+			<animation effect="slide" start="0,0" end="55,0" time="0" condition="!VideoPlayer.HasMenu | VideoPlayer.Content(LiveTV)">Conditional</animation>
```

Comment:
this one was a bit dicey but I managed to get it working only like this, else the subtitles flyout misaligned.